### PR TITLE
Jetpack section: add Automated Transfer step for Business sites

### PR DESCRIPTION
**Pull request by rcanepa**
#### Changes proposed in this Pull Request

* Show non-Atomic Business sites a CTA to Automated Transfer.

Fixes 1179060693083348-as-1179702911174270

Todo:
* Make the CTA do the AT.
* Make the CTA show a progress indicator as per design.

#### Testing instructions

**Prerequisite:** non-Atomic WPCOM site with Business plan.
* Run this PR and enable the `jetpack/features-section`.
* Visit Jetpack > Backup.
* Verify that you see the Automated Transfer CTA and that it matches the design.
* Visit Jetpack > Scan.
* Verify that you see the Automated Transfer CTA and that it matches the design.

##### While trying to access the Jetpack Backup section
image

##### While trying to access the Jetpack Scan section
image

### CODE REVIEW

File: client/components/jetpack/wpcom-business-at/index.tsx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+interface Props {
+	product: 'backup' | 'scan';
+}
+
+const contentPerPrimaryProduct = {
+	backup: {
+		documentHeadTitle: 'Activate Jetpack Backup now',
+		header: translate( 'Jetpack Backup' ),
+		primaryPromo: {
+			title: translate( 'Get time travel for your site with Jetpack Backup.' ),
```
Comment (a8ci18n) [inline client/components/jetpack/wpcom-business-at/index.tsx:43]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Get time travel for your site with Jetpack Backup' )` *ES Score: 10*

File: client/components/jetpack/wpcom-business-at/index.tsx [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+			image: { path: JetpackBackupSVG },
+			content: translate(
+				'Backup gives you granular control over your site, with the ability to restore it to any previous state, and export it at any time.'
```
Comment (a8ci18n) [inline client/components/jetpack/wpcom-business-at/index.tsx:46]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( 'Granular control over your site, with the ability to restore it to any previous state, and export it at any time.' )` *ES Score: 8*

File: client/components/jetpack/wpcom-business-at/index.tsx [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+			),
+			promoCTA: {
+				text: translate( 'Activate Jetpack Backup now' ),
+			},
+		},
+		secondaryPromo: {
+			title: translate( 'Jetpack Scan' ),
+			icon: 'security',
+			content: translate(
+				'Automated scanning and one-click fixes to keep your site ahead of security threats.'
```
Comment (a8ci18n) [inline client/components/jetpack/wpcom-business-at/index.tsx:56]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( 'Scan gives you automated scanning and one-click fixes to keep your site ahead of security threats.' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-business-automated-transfer)

**Comment by cleacos** [comment]
Sorry @rcanepa, by mistake I merged this branch, I thought that was okay, but then I realized that I didn't review it and it's still in progress. I've reverted it

**Pull request merged**